docs/3.0.0-beta.x/guides/registering-a-field-in-admin.md

```diff
@@ -65,12 +143,28 @@ const Wysiwyg = ({
   onChange,
   value,
 }) => {
+  const [isOpen, setIsOpen] = useState(false);
   let spacer = !isEmpty(inputDescription) ? <div style={{ height: '.4rem' }} /> : <div />;
 
   if (!noErrorsDescription && !isEmpty(errors)) {
     spacer = <div />;
   }
 
+  const handleChange = data => {
+    console.log(data);
+
+    if (data.mime.includes('image')) {
+      const imgTag = `<p><img src="${data.url}" caption="${data.caption}" alt="${data.alternativeText}"></img></p>`;
+      const newValue = value ? `${value}${imgTag}` : imgTag;
+
+      onChange({ target: { name, value: newValue } });
+    }
+
+    // Handle videos and other type of files
```

Comment:
Is this comment useful?

Reply:
Sure it is there to say that in the example I have only handle the image type so they will need to add the logic for other types